### ISSUE TITLE
crash on hotspot

### ISSUE THREAD

**Issue by hhumas** [meharbhutta/3MGWifiFT#2]
hi . it works on wifi  .but if you select hotspot it crashed . 
 you have to enable wifi then start app then it start working again with wifi

**Comment by meharbhutta** [comment]
It works good until Android 6.0 Marshmallow & crashes on hotspot is because of the different mobile vendors. It works good on Samsung devices.

**Comment by hhumas** [comment]
Yeah I try it. Can I get your Facebook or email. I've few questions related to this project. Thanks

**Comment by meharbhutta** [comment]
Why not. I would love to have conversation with you. Here it is muhammadmehar.bhutta@gmail.com
